TestFoundation/HTTPServer.swift

```diff
@@ -339,6 +339,12 @@ public class TestURLSessionServer {
 
     func getResponse(request: _HTTPRequest) -> _HTTPResponse {
         let uri = request.uri
+
+        if uri == "/upload" {
+            let text = "Upload completed!"
+            return _HTTPResponse(response: .OK, headers: "Content-Length: \(text.characters.count)", body: text)
```

Comment:
hmm, technically `text.characters.count` is not the length in bytes. It happens to be for this string in `.ascii.` but maybe we want to be more correct? `text.data(using: .ascii)...`